modules/ipi-install-node-requirements.adoc

```diff
@@ -20,10 +20,12 @@ ifeval::[{product-version} > 4.4]
 endif::[]
 
 ifndef::openshift-origin[]
-* *Latest generation:* Nodes must be of the most recent generation. Installer-provisioned installation relies on BMC protocols, which must be compatible across nodes. Additionally, {op-system-base} 8 ships with the most recent drivers for RAID controllers. Ensure that the nodes are recent enough to support {op-system-base} 8 for the `provisioner` node and {op-system} 8 for the control plane and worker nodes.
+* *Latest generation:* Nodes must be of the most recent generation. As the installer-provisioned installation relies on BMC protocols, the hardware must support IPMI cipher suite 17.
```

Comment:
```suggestion
* *Latest generation:* Nodes must be of the most recent generation. Because the installer-provisioned installation relies on BMC protocols, the hardware must support IPMI cipher suite 17.
```

Reply:
@kalexand-rh done